Dockerfile

```diff
@@ -197,10 +197,10 @@ FROM base as extensions
 RUN set -x; \
 	mkdir $MW_HOME/extensions \
 	&& cd $MW_HOME/extensions \
-	# AdminLinks (v. 0.6.1)
+	# AdminLinks (v. 0.6.2)
 	&& git clone --single-branch -b master https://gerrit.wikimedia.org/r/mediawiki/extensions/AdminLinks $MW_HOME/extensions/AdminLinks \
 	&& cd $MW_HOME/extensions/AdminLinks \
-	&& git checkout -q 60eda7201636218b80d83a637b70e5c753900e41 \
+	&& git checkout -q cb2065f7dc67475ce36210f3098d2fbab93ebd29 \
```

Comment:
It looks like the `cb2065f7dc67475ce36210f3098d2fbab93ebd29` commit of the `AdminLinks` matches version `0.6.3` while on the comment above version is listed as `0.6.2` https://github.com/wikimedia/mediawiki-extensions-AdminLinks/commit/cb2065f7dc67475ce36210f3098d2fbab93ebd29 so there is a typo on Canasta dockerfile

Could you please update the comment to be 0.6.3?